src/pybind/mgr/dashboard/frontend/src/app/shared/tests/util.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpClient } from '@angular/common/http';
```

Comment:
You don't have to create a file for `http` mocking. You can really mock the API in a good manner using `HttpTestingController`. Take a look [here](https://angular.io/guide/http#testing-http-requests) on how to use it. Also take a look at my [example](https://github.com/Devp00l/ceph/commit/5f3d6480e951c39b44009e753529125cd7e2c486) made for you.

Reply:
I thought `HttpTestingController` was for the deprecated `HttpModule`, so I never look into it.
I will implement that in all the tests.